ExtensionManifest.md

```diff
@@ -0,0 +1,88 @@
+    "type": "Workspace",
+    "url": "https://calculator.example.com",
+    "namePrefix": "CALC",
+    "fields": [{
+        "id": 1,
+        "name": "Sum",
+        "type": "Number",
+        "isOneToOne": true,
+        "codingSettings": "Write",
+        "excludeFromImport": true,
+    }],
+    "statistics": [{
+        "id": 1,
+        "name": "calculations performed"
+    }]
+}
+```
+
+## Top-Level Settings
+These settings define basic features required for every UIX.
+
+- `name` <[String]> Name of the UIX displayed in the case home tab list or documents workspace.
+- `type` <[String]> Location where the UIX will appear in Ringtail. Must be one of:
+  - `Workspace` - Documents area workspace pane
+  - `Case` - Case home page
+- `url` <[String]> URL of the web app to load.
+- `configuration` <[String]> (Optional) Payload stored by Ringtail and delivered to UIX upon initialization.
+- `description` <[String]> (Optional) Description to differentiate similar UIX displayed in the UIX admin area.
+- `privateKey` <[String]> (Optional) Key used to sign a JWT for authenticating Ringtail with backend UIX systems. See [security considerations](README.md#security-considerations) for more information.
+- `namePrefix` <[String]> (Optional) Prefix used to group and differentiate UIX fields and statistics from others in Ringtail. This value is required if fields or statistics are specified. **NOTE: This value cannot be changed once set.**
+- `fields` <[Array]<[Field](#fields)>> (Optional) Fields to create in each case the UIX is assigned to.
+- `statistics` <[Array]<[Statistic](#statistics)>> (Optional) Statistics counters to create in each case the UIX is assigned to.
+
+## Fields
+Providing field settings in an extension manifest allows Ringtail to automatically create fields in cases where it is installed. This is a great way to expose UIX states to Ringtail, allowing users to search, list, and view these fields.
+
+- `id` <[Number]> Identifier for this field in the manifest, used to lookup its concrete identifier for coding.
+- `name` <[String]> Name of the field. This will be prefixed with the `namePrefix` value in Ringtail.
+- `type` <[String]> Type of data the field will store. Must be one of:
+  - `YesNo` - Simple boolean value
+  - `Number` - Numeric value in the range [-99999999999.9999, 99999999999.9999] with at most four decimal digits
+  - `Date` - Date value in ISO 8601 UTC format
+  - `Text` - Simple string values with at most 255 characters
+  - `Memo` - Larger string values supporting simple HTML markup with at most 4095 characters
+  - `PickList` - Value picked from a provided list of choices
+- `isOneToOne` <[Boolean]> (Default `true`) True to only allow a single value per document, otherwise multiple, unique values may be coded. `YesNo` fields require `isOneToOne=true`.
+- `excludeFromSearch` <[Boolean]> (Default `false`) True to suppress this field from Ringtail's advanced search builder.
```

Comment:
What do you think of this pattern for Booleans: "If `true`, suppresses this field..."